### DISCUSSION TITLE
Have you seen the Sunbird Obsrv microsite yet?

### DISCUSSION THREAD

**Discussion by rhwarrier** [Sunbird-Obsrv/Community#4] [Documentation]
All,
Say hello to the **[Obsrv microsite](https://obsrv.sunbird.org)** , if you haven’t already.
You can find the best deals on clothing, electronics and home appliances, with special extra discounts for developers who can solve coding puzzles!!

Okay, maybe not - but there’s plenty you can find out about the Obsrv building block, and isn’t that incentive enough?

What we’d like to hear from you is:
1. Do you find it useful/ find it easy to get to what you need?
2. Are there specific aspects you like/ dislike/ think need better presentation?
3. Did you find anything that should have been there, but isn’t?

In true open source spirit, we’d like to ensure that this microsite is ‘of the people, by the people and for the people’

While internal feedback is great, we’d also welcome thoughts from our partners and adopters who may not have seen the site yet. Sometimes, it’s just a fresh pair of eyes that can make all the difference.

Please share your thoughts and pointers **[here](https://github.com/Sunbird-Obsrv/Community/discussions/categories/documentation)** for us to take forward. We will iterate continuously to keep improving the site. 

[@Krishnaj20](https://github.com/Krishnaj20)  [@pramodkvarma](https://github.com/pramodkvarma) [@jagadishbabu](https://github.com/jagadishbabu) [@madhucr](https://github.com/madhucr) [@alok-os](https://github.com/alok-os) [@surendrasinghs](https://github.com/surendrasinghs) [@kameshbhr](https://github.com/kameshbhr) [@SanthoshVasabhaktula](https://github.com/SanthoshVasabhaktula) [@vrayulu](https://github.com/vrayulu) [@mmanimegalai](https://github.com/mmanimegalai) [@maheshkumargangula](https://github.com/maheshkumargangula) [@reshmi-nair](https://github.com/reshmi-nair) [@sharathkashyap](https://github.com/sharathkashyap) [@vinukumar-vs](https://github.com/vinukumar-vs) [@gandham-santhosh](https://github.com/gandham-santhosh) [@anandp504](https://github.com/anandp504) [@chitranshu-keshav](https://github.com/chitranshu-keshav) [@Prerna789](https://github.com/Prerna789) [@coolbung](https://github.com/coolbung) [@vijiurs](https://github.com/vijiurs) [@aks30](https://github.com/aks30) [@kiranharidas187](https://github.com/kiranharidas187) [@aayushkedia95](https://github.com/aayushkedia95) [@ChakshuGautam](https://github.com/ChakshuGautam) [@vaivk369](https://github.com/vaivk369) [@ahghatol](https://github.com/ahghatol) [@Nadiya496](https://github.com/Nadiya496) [@ashokreddy1208](https://github.com/ashokreddy1208) @rhwarrier 

A few things that are being worked upon already:
> Additional API swagger documentation will be moved to the Obsrv microsite from sunbird.org
> Further details are being added relating to the installation of individual components

Thanks,
- team Obsrv

**Comment by pramodkvarma** [comment]
Excellent 👏👏👏!! Will be going through. Wherever I can update myself, I will, otherwise, will report it.
